### ISSUE TITLE
Will the remastered version of CTC be released?

### ISSUE THREAD

**Issue by kir12** [danmaq/touhou-ctc-danmakufu#1]
Hi, I played through the Danmakufu version of CTC and while the game *itself* was great, I had a number of technical difficulties with Danmakufu. I read that there was a version of CTC [remastered using Microsoft XNA](https://en.touhouwiki.net/wiki/Concealed_the_Conclusion) released back in 2011, and I was wondering if you had any plans to release that as well. 

EDIT: a Google-Translated Japanese version of my question:

こんにちは、私はCTCのDanmakufuバージョンを通してプレーしました、そして、ゲーム自体が素晴らしかった間、私はDanmakufuに関して多くの技術的な困難を経験しました。 2011年にリリースされた[Microsoft XNA](https://en.touhouwiki.net/wiki/Concealed_the_Conclusion))を使用してリマスターされたCTCのバージョンがあることを私は読みました、そしてあなたがそれを同様にリリースする計画があるかどうか疑問に思いました。

**Comment by danmaq** [comment]
Thank you for issue. Ours answer is __yes. But, it may be incomplete__.
Following is a slightly longer descriptions.

We used to plan to run CtC natively on Windows, it called v3.NET. And started development based on XNA 3.1 in 2009, actually built up to about 60% completeness. However, due to the increase in the number of development costs and the future of XNA becoming unclear (In fact, XNA is obsoleted), develop was pended. There is no prospect of continuing.

The source code of this project will be released in the nearly future with a CC-BY 4.0 license. However, resources such as images and songs will be licensed under CC-BY-NC 4.0. At present, there are several reasons for delay this:

- Migration to Git fails due to large files.  
This project is managed by Subversion, and there is a dump of a repository with a capacity of 7.6GB with bzip2 compression. I tried to migrate this repository to Git but it failed. The cause is that it contains 2GB class binary files inside (packed audio with XACT).
- Inability to build due to lack of some source code, and dependency on old build environment
As you know, this project relies on Microsoft XNA 3.1. And inevitably it depends on .NET Framework 3.5 and Visual Studio 2008. Also, the structure of this project requires the VS2008 Standard Edition or higher. However, since these environments do not exist, it cannot be built from now on. There are already built binaries, but I have not confirmed their working correctly.

In other words, it is not impossible to release within a few days if it is a condition that excludes large files that require version history and LFS. However, I did not feel the need to release it as soon as possible, so I was planning to wait for the solution of the problem and then release it.

**Comment by nixls** [comment]
Hey, are there any updates on this?

**Comment by kohy-creates** [comment]
Are there any updates on this now?

**Comment by kohy-creates** [comment]
maybe now?

**Comment by doremyy** [comment]
bump?

**Comment by abysswatched** [comment]
I hope to see this one day, CtC is one of my favorite touhou fanmade games
